src/main/scala/scalatutorial/sections/StandardLibrary.scala

```diff
@@ -73,7 +73,7 @@ object StandardLibrary extends ScalaTutorialSection {
    *
    * === Right Associativity ===
    *
-   * Convention: Operators ending in “`:`” associate to the right.
+   * Convention: Operators ending in “`::`” associate to the right.
```

Comment:
the original text is right, it applies to `::` but also others like `+:` and `++:`